documentation/source/user_section/installation/linux.rst

```diff
@@ -145,7 +145,14 @@ In the context of SCT, it can be used:
 Basic Installation (No GUI)
 ***************************
 
-First, `install Docker <https://docs.docker.com/install/>`_. Then, follow the examples below to create an OS-specific SCT installation.
+First, `install Docker <https://docs.docker.com/engine/install/>`_.
+
+.. note::
+   Docker Desktop for Linux is not recommended if you intend to use the GUI.
+   Instead install the `Docker CE Engine <https://docs.docker.com/engine/install>`_, which is separate to the Docker Desktop Engine.
+   For example on Ubuntu/Debian, follow the instructions for installing Docker from the `apt repository <https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository>`_.
+
+Then, follow the example below to create an OS-specific SCT installation.
```

Comment:
I tried the instructions again from scratch with Docker Server, and realized that the commands in the example need `sudo`, unless the user [creates the `docker` group](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).

We might want to add another section (between "install docker" and "run the docker steps") that specifically mentions the `sudo` workaround?

```suggestion
By default, Docker commands require the use of ``sudo`` for additional permissions. If you want to run Docker commands without needing to add ``sudo`` (as shown in the example below), please follow `these instructions <https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user>`_ to create a Unix group called ``docker``, then add users your user account to it.

Then, you can follow the example below to create an OS-specific SCT installation.
```